src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.vb

```diff
@@ -630,11 +630,7 @@ End Class]]>.Value
 
             Dim expectedOperationTree = <![CDATA[
 IInvalidStatement (OperationKind.InvalidStatement, IsInvalid) (Syntax: 'Case x')
-  Children(1):
```

Comment:
We do care about this for analyzers who want to run in presence of errors. By stripping of the entire operation tree here, analyzers such as remove unused parameters analyzer will fire in this method, but will not do so with the original code. I don't think we should strip of subtrees of IInvalidStatement/IInvalidExpression - we did a bunch of work to add these children to avoid false reports.